sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -44,6 +44,7 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper wit
       pushDownFilters,
       pushDownAggregates,
       pushDownLimits,
+      pushDownOffsets,
```

Comment:
shall we push down offset before limit? The natural operators order in SQL is offset then limit.

Reply:
OK

Reply:
according to the doc, OFFSET should be pushed after LIMIT now.

Reply:
Or can we push them down in one step?